translations/round_structure.tex/pl.po

```diff
@@ -80,6 +80,27 @@ msgid ""
 "Keep playing new Rounds until any of the Scenario's ending conditions have been met.\n"
 "\n"
 msgstr ""
+"Rozgrywka podzielona jest na Rundy, podczas których każdy gracz wykonuje swoją Turę w kolejności zgodnej z ruchem wskazówek zegara, zaczynając od gracza rozpoczynającego grę.\n"
+"W swoich Turach gracze przemieszczają swoich \\hyperlink{Bohater}{Bohaterów} na Mapie Gry, budują nowe Budynki w swoich \\hyperlink{Miasto}{Miastach} i rekrutują \\hyperlink{Units}{Jednostki}, próbując spełnić warunek zwycięstwa Scenariusza.\\par\n"
+"\\textbf{Na początku każdej Rundy}, z wyjątkiem pierwszej, wykonaj następujące kroki:\n"
+"\\begin{itemize}\n"
+" \\item Obróć na aktywną stronę wszystkie użyte Żetony Budowy, Populacji i/lub Księgi Zaklęć.\n"
```

Comment:
co myślicie o takim uproszczeniu i usunięciu `/lub` - imo tu nie trzeba rozróżnienia logicznego i obracamy wszystko na aktywną stronę u każdego gracza
 
```suggestion
" \\item Obróć na aktywną stronę użyte Żetony Budowy, Populacji i Księgi Zaklęć.\n"
```

Reply:
Brzmi to lepiej, jestem za